docs/develop/integrations/vercel-integration.md

```diff
@@ -0,0 +1,93 @@
+
+![Create cache form](/img/console-create-cache-form.png)
+
+After pressing the `Create` button you'll see the new `chat` cache in the list of available caches.
+
+![Chat cache](/img/console-caches-chat.png)
+
+Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure that you choose the same region when you generate a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, choose the cloud provider and region you used to create your cache, choose the `Super User Token` token type, and hit the `Generate Token` button.
+
+![Generate token](/img/console-generate-token-form.png)
+
+Copy the `Auth Token` and save it in a safe place. You'll need to use it later to configure your Vercel deployment, where you will add it as an environment variable for use in the chat app.
+
+![Generated token](/img/console-generate-token-result.png)
+
+## Deploying with Vercel
+
+Now it's time to log into Vercel to configure and deploy the application. As noted before, you need a copy of the Momento Javascript SDK available in your repository. In this example, we'll deploy from a GitHub repository forked from [the original](https://github.com/momentohq/client-sdk-javascript).
+
+![Fork SDK Repository](/img/github-fork-js-sdk.png)
+
+After logging into Vercel, navigate to your `Overview` page and press the `Create a New Project` button.
+
+![Vercel overview page](/img/vercel-overview.png)
+
+From the list of repositories available to you, press the `Import` button for `client-sdk-javascript`.
+
+![Vercel import repository](/img/vercel-import-repository.png)
+
+The next screen allows you to configure your deployment. Because we are deploying an example application from a subdirectory of the Momento Javascript SDK, you'll need to set the `Root Directory` in the `Configure Project` section of the page. The path to the chat example is `/examples/web/nextjs-chat`, so you'll enter that as the `Root Directory`, using the `Edit` button to navigate to `examples` -> `web` -> `nextjs-chat`.
+
+![Vercel root directory](/img/vercel-root-directory.png)
+
+Notice the Next.js icon next to the selected directory. Vercel has examined the contents of the directory and detected that our project is using Next.js. Because of this auto-detection, we can simply leave the `Build and Output Settings` section alone. It will select default values appropriate to our project. 
+
+Recall that we generated and saved a Momento auth token earlier for use with this application. Use the `Environment Variables` section below the build settings to add an environment variable with `MOMENTO_AUTH_TOKEN` as the name and the token you generated as the value. Next, add a second environment variable with `NEXT_PUBLIC_MOMENTO_CACHE_NAME` as the key and `chat` as the value to point your app to the cache you created earlier.
```

Comment:
Is there no secure storage for things in Vercel that are kind of like AWS Secrets Manager? If it is possible to have examples using environment variables and using a secrets manager, that'd be best practice.

Reply:
this is a good call-out but I think we can circle back and improve it in a subsequent iteration whenever someone has time to do a bit of research on vercel secrets.